spec/index.bs

```diff
@@ -2030,6 +2030,9 @@ that returns a new buffer in the [=buffer state/mapped=] or [=buffer state/unmap
 <script type=idl>
 [Exposed=(Window, DedicatedWorker), SecureContext]
 interface GPUBuffer {
+    readonly attribute GPUSize64 size;
+    readonly attribute GPUBufferUsageFlags usage;
```

Comment:
needs to be `unsigned long`; `[EnforceRange]` is not allowed on readonly attributes: https://webidl.spec.whatwg.org/#EnforceRange

Reply:
Good catch, thanks! Updated all of them.